libraries/swapfile_provider.rb

```diff
@@ -89,7 +89,7 @@ def swapoff
       end
 
       def remove_swapfile
-        ::FileUtils.rm(@new_resource.path)
+        ::FileUtils.rm_f(@new_resource.path)
```

Comment:
I think we should leave this as-is. If this function is called and the swap does not exist, I think it should fail. We should guard against calling this function higher up, as you did above. Thoughts?